core/DataTable/Renderer.php

```diff
@@ -265,7 +265,7 @@ protected function getApiMetaData()
 
             $api = \Piwik\Plugins\API\API::getInstance();
             $meta = $api->getMetadata($this->idSite, $apiModule, $apiAction);
-            if (is_array($meta[0])) {
+            if (!empty($meta[0]) && is_array($meta[0])) {
```

Comment:
Should this be `!isset($meta[0])` in case `$meta[0]` is `[]`? Not sure if it should be noticed it would be ignored in that case.

Reply:
sure. isset makes sense
```suggestion
            if (isset($meta[0]) && is_array($meta[0])) {
```